source/common/runtime/runtime_features.cc

```diff
@@ -115,6 +115,9 @@ FALSE_RUNTIME_GUARD(envoy_reloadable_features_always_use_v6);
 FALSE_RUNTIME_GUARD(envoy_reloadable_features_enable_include_histograms);
 // TODO(wbpcode) complete remove this feature is no one use it.
 FALSE_RUNTIME_GUARD(envoy_reloadable_features_refresh_rtt_after_request);
+// TODO(adip) this will be enabled by default once the work on the feature is
+// done in EdsClusterImpl.
+FALSE_RUNTIME_GUARD(envoy_restart_features_use_eds_cache_for_ads);
```

Comment:
Please add to release notes coverage of this feature and runtime guard.

Reply:
The release note will be added once the feature is fully integrated into EDS (next PR will include the changes needed in the EDS subscription - see [the entire feature implementation](https://github.com/adisuissa/envoy/commit/f0b7ac8ab85776947a8332efe1c3759e8714b9ed)).